templates/expo-template-tabs/__tests__/App-test.js

```diff
@@ -1,29 +1,34 @@
-import * as React from 'react';
-import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
-import renderer from 'react-test-renderer';
 
+import * as React from 'react';
+import { act, create } from 'react-test-renderer';
 import App from '../App';
 
 jest.mock('expo', () => ({
-  AppLoading: 'AppLoading',
+  Linking: {
+    makeUrl: () => '/',
+  },
+  SplashScreen: {
+    preventAutoHide: () => 'preventAutoHide',
+    hide: () => 'hide',
+  },
 }));
 
-jest.mock('../navigation/AppNavigator', () => 'AppNavigator');
+
+jest.mock('shortid', () => jest.fn(() => '1'))
```

Comment:
this is throwing an error for me, but removing the line seems to resolve it. Why do we need to mock `shortid`?

Reply:
@cruzach what error do you get? Because the updates were done per your feedback on the underlying issue that backs this pr, more specifically #7155 . Also the mock of shortid is per the feedback obtained from the issue linked here for the `react-navigation` package. Not mocking it would make it impossible for the snapshots to be correct. As the key prop in the `BottomTabNavigator` would be diferent at every run and with that the user would be prompted to update the snapshot at each iteration. If you check the commited snapshot file and look for `BottomTabNavigator` you'll see it in action.

Reply:
hm that's weird, I'm not getting it now. I must have had something wrong on my end

Reply:
making these changes in a blank `expo init` tabs project gives me:
```
Cannot find module 'shortid' from 'App-test.js'
```

Reply:
@cruzach i've finished initializing a new project using the template and grabbed the code i was using and you're correct. It seems that `react-navigation` decided to drop `shortid` in favor of `nanoid` for their key prop implementation. With that i've been trying to adjust the test to mock the package to no avail. If you have any ideas let me know. Otherwise i'll just update the test to make it more simple so that a user can have tests available at boot.

Feel free to provide feedback

Reply:
> Otherwise i'll just update the test to make it more simple so that a user can have tests available at boot.

this sounds good to me

Reply:
@cruzach with the latest commit i've simplified the test. Not the ideal setup but at least it gets the tests working. Feel free to provide feedback.

Reply:
I'll try to find time to dig into this.. I'd rather not simply check if the JSON is truthy, otherwise why have the snapshot, right?